lib/diagnostic.js

```diff
@@ -7,13 +7,16 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function (/* */) {
-  /* your response here */
+  const normalizeWords = normalizeWords.split(' ')
```

Comment:
This function should accept a parameter, and then execute `.split`on the passed in argument. Something like this: `normalizeWords(string){ string.split(' ')`